src/event/event.jl

```diff
@@ -0,0 +1,78 @@
+"""
+    event()
+Returns the event.json content as a Dict.
```

Comment:
Have you considered returning a `WebhookEvent` from `GitHub.jl` instead of a dict?

Reply:
It makes sense. But I was not sure if we want to add that dependency.